### ISSUE TITLE
torch.empty_strided throws core dump

### ISSUE THREAD

**Issue by ailzhang** [pytorch/xla#881]
```
import torch
import torch_xla

device = 'xla:0'

shape = (2, 3, 4)
strides = (12, 4, 1)
xla_result = torch.empty_strided(shape, strides, device=device)

print(xla_result)
```
I guess it's okay if we don't support it, we can fix the core dump with a proper error.

**Comment by ailzhang** [comment]
https://github.com/pytorch/pytorch/issues/23688

**Comment by ailzhang** [comment]
This was fixed by #938, closing
